app/services/profiles/edit_service.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Profiles
+  class EditService < ApplicationService
+    attr_reader :user
+
+    def initialize(profile, params)
+      super()
+      @profile = profile
+      @params  = params
+    end
+
+    def call
+      @profile.update(@params)
```

Comment:
please add service and system specs. 
You and I will be sure that everything is working correctly then

Reply:
where are spec? 👀👀👀👀👀👀👀